core/templates/dev/head/I18nFooter.ts

```diff
@@ -21,13 +21,12 @@
 require('services/translation-file-hash-loader.service.ts');
 
 angular.module('oppia').directive('i18nFooter', [
-  'UrlInterpolationService', function(UrlInterpolationService) {
+  function() {
     return {
       restrict: 'E',
       scope: {},
       bindToController: {},
-      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
-        '/i18n-footer.directive.html'),
+      template: eval(require('html-loader!./i18n-footer.directive.html'))(),
```

Comment:
@vojtechjelinek `eval` is used here and elsewhere because requiring the files using html-loader returns a string containing the a function that returns the content of the file.